index.html

```diff
@@ -12,14 +12,64 @@
     />
   </head>
   <body>
+    <section class="popup" id="popup">
+      <div class="modal">
+        <div class="modal-head">
+          <h1 id="post-title">Title Here</h1>
+          <button type="button" class="close_btn">
+            <i class="fa fa-times"></i>
+          </button>
+        </div>
+        <div class="modal-content">
+          <img id="post-image" src="images/Snapshoot_Portfolio.png" alt="work image" />
+          <div class="cont1">
+            <p id="post-text">
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s with the releaLorem Ipsum is simply
+              dummy text of the printing and typesetting industry. Lorem Ipsum
+              has been the industry's standard dummy text ever since the 1500s,
+              when an unknown Lorem Ipsum is simply dummy text of the printing
+              and typesetting industry. Lorem Ipsum has been the industry's
+              standard dummy text ever since the 1500s, when an unknown printer
+              took a galley of type and scramble.
+            </p>
+            <ul id="post-techno">
+              <li><a href="">css</a></li>
+              <li><a href="">html</a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href="">Ruby</a></li>
+            </ul>
+            <div class="modal-btn">
+              <button type="button" class="btn">
+                See Live&nbsp;&nbsp;<i class="fa-solid fa-power-off"></i>
+              </button>
+              <button type="button" class="btn">
+                See Source&nbsp;&nbsp;<i class="fa-brands fa-github"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
Nice job with the popup window 👍. But as the projects require the `pop up` window should render the project detail dynamically from the `JavaScript object` where the object contains the information for each project 👇.

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/34578631/168042039-304479f1-1ada-457a-85bf-56ca5ce43d1f.png">

Kindly, remove this `hardcoded` data from the `index.html` file and get the data of each project from the JavaScript object and render it to the page.